test/models/user_test.rb

```diff
@@ -351,6 +351,55 @@ class UserTest < ActiveSupport::TestCase
     assert_equal 3, u.editable_theses.count
   end
 
+  test 'editable_theses returns a thesis with neither metdata nor issues flags set' do
```

Comment:
typo `metdata` vs  `metadata`

Reply:
Ah, good catch. I'll fix and rebase before merging.